.gitmodules

```diff
@@ -1,6 +0,0 @@
-[submodule "hs_docker_base"]
```

Comment:
@cuahsimarko @cbcunc @engineSound.  With this PR @hyi is removing hs_docker_base and hs_postgres as submodules of hydroshare because we're not actually using the code, rather pulling in docker images.  I'm fine with this change, but wanted to tag everyone in case others disagree.

Reply:
Thanks for calling this out Scott. I believe new developer setup will be unaffected, but if anything needs to be updated, I can take care of that in 1.21. To my knowledge, @martinseul  and I are the only people using new developer setup.

Reply:
@sblack-usu Thanks for including me.